__tests__/app.test.js

```diff
@@ -122,6 +122,73 @@ describe("/api/articles", () => {
           });
         });
     });
+
+    test("GET 200: ?sort_by=created_at&order=asc returns articles sorted by created_at in ascending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=created_at&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles.length).toBeGreaterThan(0);
+          expect(body.articles).toBeSortedBy("created_at", {
+            descending: false,
+          });
+        });
+    });
+
+    test("GET 200: ?sort_by=votes&order=desc returns articles sorted by votes in descending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes&order=desc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles.length).toBeGreaterThan(0);
+          expect(body.articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+
+    test("GET 200: ?sort_by=comment_count&order=asc returns articles sorted by comment_count in ascending order", () => {
+      return request(app)
+        .get("/api/articles?sort_by=comment_count&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles.length).toBeGreaterThan(0);
+          expect(body.articles).toBeSortedBy("comment_count", {
+            descending: false,
+          });
+        });
+    });
+
```

Comment:
I'm not sure that this test needed to happen ^^ you've already proven that a valid column can be sorted by asc :)

Reply:
Actually, this made me realise that I didn't make sure that the sort_by and order resolves to defaults, if nothing is passed. I made an error handler for that instead. :sweat_smile: Gonna fix this in my next commit. :raised_hands::skin-tone-3: Thanks Niamh!